<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieRendererWindow</class>
 <widget class="QMainWindow" name="MovieRendererWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QML Movie Renderer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>QML File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qmlFileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="openQmlFileButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Output Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="widthSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4096</number>
         </property>
         <property name="value">
          <number>1280</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="heightSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4096</number>
         </property>
         <property name="value">
          <number>720</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Duration (seconds)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="durationSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Frames per Second</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fpsSpinBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1200</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ouput Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputDirectoryLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="outputDirectoryPushButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Output Filename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputFilenameLineEdit">
         <property name="text">
          <string>output</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Image Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="imageFormatComboBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="renderMovieButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Render Movie</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
